internal/modules/list.go

```diff
@@ -104,6 +105,57 @@ func List(ctx context.Context, client kube.Client) (ModulesList, error) {
 	return modulesList, nil
 }
 
+// ListCatalog returns list of module catalog on a cluster
+// collects info about modules based on ModuleTemplates and ModuleReleaseMetas
+func ListCatalog(ctx context.Context, client kube.Client) (ModulesList, error) {
+	moduleTemplates, err := client.Kyma().ListModuleTemplate(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to list all ModuleTemplate CRs from the cluster")
+	}
+
+	modulereleasemetas, err := client.Kyma().ListModuleReleaseMeta(ctx)
```

Comment:
Why have you used flat case for this variable instead of camelCase which we rather use?

Reply:
missclick :) I'm fixing it [the another PR](https://github.com/kyma-project/cli/pull/2343/commits/48510ceef0c08abfaf29062d9b10b8b30701eb72#diff-8cad00926d76716bf91b66d28d39976ff47230dd548df0cab809cc357ed60850L116)